src/default_tests.jsonc

```diff
@@ -16,17 +16,18 @@
     // uncomment the second to run NOAA-GFDL-AM4 only for the MJO_prop_amp POD,
```

Comment:
New PODs shouldn't modify (or need to modify) framework code in `/src`. Please commit the changes from `git checkout develop -- src` to bring the contents of `/src` back to what's in the develop branch.

Reply:
Done afb8f0a